src/chains.rs

```diff
@@ -26,14 +26,20 @@ use crate::device::Device as DeviceAPI;
 use crate::{ContextID, Error, SurfaceAccess, SurfaceInfo, SurfaceType};
 use euclid::default::Size2D;
 use fnv::{FnvHashMap, FnvHashSet};
+use glow as gl;
+use glow::HasContext;
 use log::debug;
-use sparkle::gl::{self, GLuint, Gl};
 use std::collections::hash_map::Entry;
 use std::fmt::Debug;
 use std::hash::Hash;
 use std::mem;
+use std::num::NonZero;
 use std::sync::{Arc, Mutex, MutexGuard, RwLock, RwLockReadGuard, RwLockWriteGuard};
 
+fn frame_buffer(frame_buffer: u32) -> Option<glow::NativeFramebuffer> {
```

Comment:
I think it makes sense to make this a `fn framebuffer()` getter on `impl Surface`. Note that `framebuffer` is spelled without the space in other parts of `surfman`.

Reply:
> I think it makes sense to make this a fn framebuffer() getter on impl Surface

Er, what? glow is limited to chains.rs file and this function is only used to prepare input arguments for glow.